LEGO1/lego/legoomni/src/entity/legoworldpresenter.cpp

```diff
@@ -359,8 +359,8 @@ MxResult LegoWorldPresenter::FUN_100674b0(ModelDbModel& p_model, FILE* p_wdbFile
 	action.SetDirection(p_model.m_direction);
 	action.SetUp(p_model.m_up);
 
-	MxU32 objectId = m_unk0x50;
-	m_unk0x50++;
+	MxU32 objectId = m_maxObjectId;
+	m_maxObjectId++;
```

Comment:
Given this logic, I think a better name would be `m_nextObjectId` - it uses the current value for the next object, then increments it to get a new ID for the next object and so on

Reply:
Good point, changed it.